recipes/base_install.rb

```diff
@@ -66,18 +70,18 @@
   end
 end
 
-case node['platform_family']
-when 'rhel', 'amazon'
-  yum_package node['cfncluster']['kernel_devel_pkg']['name'] do
-    version node['cfncluster']['kernel_devel_pkg']['version']
-    retries 3
-    retry_delay 5
-  end
-when 'debian'
-  apt_package node['cfncluster']['kernel_generic_pkg'] do
-    retries 3
-    retry_delay 5
+package "install kernel packages" do
```

Comment:
It's an interesting change to mention in the commit message. 
Great idea to use `package` instead of the specific `yum_package` and `apt_package`.

Let's double check the entire code to see if there are other places to change.